src/components/skip-button/SkipButton.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { Alert, Pressable, PressableProps } from 'react-native'
+import TrackPlayer, { useProgress } from 'react-native-track-player'
+import Icon from 'react-native-vector-icons/Feather'
+
+import { JumpInterval } from '@constants'
+import { AppColors, Spacing } from '@theme'
+
+interface ISkipButtonProps {
+  forward: boolean
+}
+const SkipButton = (props: ISkipButtonProps) => {
```

Comment:
please add spaces after every common declarations. For eg: after all the states, skip a line and then declare something else.